README.md

```diff
@@ -15,150 +15,67 @@ ILP wallet with hosted ledger and connector instances
-`API_TRACK_MIXPANEL` | | Mixpanel Tracking ID.
-`API_GITHUB_CLIENT_ID` | | Github application client id (used for github oauth).
-`API_GITHUB_CLIENT_SECRET` | | Github application client secret (used for github oauth).
-`API_MAILGUN_API_KEY` | | Mailgun api key (for sending emails).
-`API_MAILGUN_DOMAIN` | `wallet.com` | One of the domains attached to the Mailgun account.
-`API_ANTIFRAUD_SERVICE_URL` | `antifraud.wallet.com` | Anti fraud service url. This will enable an additional step in registration that asks for personal details
-`API_ANTIFRAUD_MAX_RISK` | `20` | Maximum tolerable risk level for the registration
-`API_EMAIL_SENDER_NAME` | `info` | Email sender name
-`API_EMAIL_SENDER_ADDRESS` | `contact@wallet.com` | Email sender address
-`API_REGISTRATION` | `true` | Enable/Disable registration
-`WALLET_FORCE_HTTPS` | `true` | Force all connections to use HTTPS.
-`WALLET_TRUST_XFP_HEADER` | `true` | Trust the `X-Forwarded-Proto` header.
-`CONNECTOR_ENABLE` | `false` | Run a connector instance
-`CLIENT_PUBLIC_PORT` | `80` | Client public port (if different from `CLIENT_PORT`)
-`CLIENT_TITLE` | `ILP Kit` | Browser title and logo
-
-##### Default five-bells-ledger environment variables 
-(used if the `API_LEDGER_URI` is not specified). You can read more about these variables in the [five-bells-ledger readme](https://github.com/interledgerjs/five-bells-ledger#step-3-run-it).
-
-Name | Default |
----- | ------- |
-`LEDGER_DB_URI` | `API_DB_URI`
-`LEDGER_ADMIN_USER` | `API_LEDGER_ADMIN_USER`
-`LEDGER_ADMIN_PASS` | `API_LEDGER_ADMIN_PASS`
-`LEDGER_HOSTNAME` | `API_HOSTNAME`
-`LEDGER_PORT` | `API_PORT + 1`
-`LEDGER_PUBLIC_PORT` | `CLIENT_PORT`
-`LEDGER_PUBLIC_PATH` | `ledger`
-`LEDGER_CURRENCY_CODE` | `USD`
-`LEDGER_CURRENCY_SYMBOL` | `$`
-`LEDGER_PUBLIC_HTTPS` | `API_PUBLIC_HTTPS`
-
-## Advanced Mode
-ILP kit UI comes with an "advanced mode" for developers and advanced users. You can activate it with a hot-key: `option+d` on Mac or `alt+d` on Windows. 
-
-## Architecture
-ILP kit consists of a [Node.js](https://github.com/nodejs/node) (developed on v6.9.1) backend (REST API) and a client built using [React](https://github.com/facebook/react).
-
-### Backend (REST API)
-The backend is responsible for communicating with the ILP ledger, creating accounts, sending payments and keeping the payment history.
-
-#### API docs
-[http://interledger.org/ilp-kit/apidoc](http://interledger.org/ilp-kit/apidoc/)
-
-#### SPSP
-
-The wallet implements [SPSP](https://github.com/interledger/rfcs/blob/master/0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md) for initiating and receiving payments.
-
-##### Webfinger
-Webfinger is used to lookup account/user identifiers.
-
-Example request 
-```bash
-curl -X GET
-https://wallet.example/.well-known/webfinger?resource=acct:alice@wallet.example
+# Quick Start
+
+_Prerequisites: Git, Node v6.9.1, `build-essentials` (or XCode command-line tools on OSX)_
```

Comment:
It would be good to provide a link to documentation for installing the xcode tools